python/tests/toolchains/run_acceptance_test.py

```diff
@@ -14,23 +14,49 @@
 
 import os
 import subprocess
+import sys
+import tempfile
 import unittest
 
 
 class TestPythonVersion(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        os.chdir("%test_location%")
+        python_version_test_dirname = os.path.dirname(
+            os.path.realpath("python_version_test.py")
+        )
+        rules_python_path = os.path.normpath(
+            os.path.join(python_version_test_dirname, "..", "..", "..", "..")
+        )
+
+        is_windows = "win" in sys.platform
+        if is_windows:
+            newline = "\r\n"
+            os.environ["HOME"] = tempfile.mkdtemp()
```

Comment:
We can use `TEST_TMPDIR` provided by Bazel.

Reply:
Yeah, that'd be much better since it should show up in sandbox paths, making things more debuggable.

Reply:
```suggestion
            home = os.path.join(os.environ["TEST_TMPDIR"], "HOME")
            os.mkdir(home)
            os.environ["HOME"] = home
```